src/LibraryInstaller.Vsix/LibraryInstallerPackage.cs

```diff
@@ -43,7 +43,9 @@ public sealed class LibraryInstallerPackage : AsyncPackage
         {
             if (await GetServiceAsync(typeof(IMenuCommandService)).ConfigureAwait(false) is OleMenuCommandService commandService)
             {
-                InstallLibraryCommand.Initialize(this, commandService);
+                #if UI_ENABLED
```

Comment:
nit: Indentation looks off here.  Usually #if directives are not indented and their contents are left at a normal indentation level.  That's what I see also when I do Format Document on this file.